src/Migration/Sources/CSV.php

```diff
@@ -180,6 +180,9 @@ private function exportDocuments(int $batchSize): void
             }
         }
 
+        $manyToManyKeys = \array_flip($manyToManyKeys);
+        $arrayKeys = \array_flip($arrayKeys);
```

Comment:
we can save this step by just adding the key directly into the var, right?
`$manyToManyKeys[$key] = true;`, etc.